libbeat/docs/shared-ilm.asciidoc

```diff
@@ -1,138 +1,114 @@
-
-. Enable index lifecycle management by setting `setup.ilm.enabled: true`. For example:
-+
---
-[source,yaml]
-------------------------------------------------------------------------------
-setup.ilm.enabled: true
-------------------------------------------------------------------------------
-
-This configuration overwrites your index settings and adjusts the {beatname_uc}
-template to use the correct settings for index lifecycle management.
-
-NOTE: If you've previously loaded the index template for this version into {es}, 
-you must overwrite the template by setting `setup.template.overwrite: true`.
-
-The rollover alias is set to +{beatname_lc}-\{{beat_version_key}\}+ by default. You
-can change the prefix used in the alias by setting `ilm.rollover_alias`, but you
-can't remove +{beat_version_key}+ from the rollover alias name. The default pattern
-used for the rollover index is `%{now/d}-000001`. You can change the
-pattern by setting `ilm.pattern`. For example:
+== Configure index lifecycle management
+
+Use the {ref}/getting-started-index-lifecycle-management.html[index lifecycle
+management] feature in {es} to manage your {beatname_uc} indices as they age.
+For example, instead of creating daily indices where index size can vary based
+on the number of Beats and number of events sent, use an index lifecycle policy
+to automate a rollover to a new index when the existing index reaches a
+specified size or age.
+
+Starting with version 7.0, {beatname_uc} uses index lifecycle management by
+default when it connects to a cluster that supports lifecycle management.
+{beatname_uc} loads the default policy automatically and applies it to any
+indices created by {beatname_uc}.
+
+You can view and edit the policy in the *Index lifecycle policies* UI in {kib}.
+For more information about working with the UI, see
+{kibana-ref}/index-lifecycle-policies.html[Index lifecyle policies].
+
+// REVIEWERS: I'm not going to describe the default policy as requested in the
+// GitHub issue because the settings might change. If users want to know what
+// the policy settings are, they should view the policy in Kibana. Sound fair?
+
+Example configuration:
 
 ["source","yaml",subs="attributes"]
 ----
-output.elasticsearch:
-  hosts: ["localhost"]
-  ilm.enabled: true
-  ilm.rollover_alias: "{beatname_lc}"
-  ilm.pattern: "{now/d}-000001" <1>
+setup.ilm.enabled: auto
+setup.ilm.rollover_alias: "{beatname_lc}"
+setup.ilm.pattern: "{now/d}-000001" <1>
 ----
 <1> Date math is supported here. For more information, see
 {ref}/indices-rollover-index.html#_using_date_math_with_the_rollover_api[Using
 date math with the rollover API].
 
-NOTE: If you modify the `rollover_alias` or `pattern` settings after loading the
-index template, you must overwrite the template to apply the changes.
---
+// REVIEWERS: Rather than showing the default for setup.ilm.pattern, let's show
+// something different than the default. What would you suggest here?
```

Comment:
Hm... I think I'd stick with the default or do not show it in the example. It's the recommended setting and I'd rather not want users to change it.
Maybe remove the 'date' part from the pattern if you want it to differ from the default.

Reply:
Leaving as-is (to show the default).